fdbserver/DDTeamCollection.h

```diff
@@ -595,6 +595,12 @@ class DDTeamCollection : public ReferenceCounted<DDTeamCollection> {
 
 	void addTeam(std::set<UID> const& team, bool isInitialTeam) { addTeam(team.begin(), team.end(), isInitialTeam); }
 
+	// FIXME: Public for testing only
+	void enableBuildingTeams(bool doBuildTeams) { this->doBuildTeams = doBuildTeams; }
```

Comment:
Since this function is called `enabled...`, but is only ever used to disable build teams, I think it would make more sense to add a method:
```
void disableBuildingTeams() { doBuildTeams = false; }
```

Reply:
Sure. Done.